src/libstd/collections/hash/table.rs

```diff
@@ -741,6 +741,9 @@ struct RawBuckets<'a, K, V> {
     marker: marker::PhantomData<&'a ()>,
 }
 
+unsafe impl<'a, K: Send, V: Send> Send for RawBuckets<'a, K, V> {}
```

Comment:
Hm, it's not obvious to me that this is the right thing. Maybe this should be `K: Sync, V: Sync`? Basically, I'm asking what reference/value semantics does `RawBuckets` have with `K` and `V`?